mira/metamodel/decpodes_examples.py

```diff
@@ -0,0 +1,40 @@
+import json
```

Comment:
This file definitely shouldn't be in the metamodel module. It belongs under examples